sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -548,34 +552,59 @@ public virtual Response ActivateMultiSlot(string eventId, CancellationToken canc
             }
         }
 
-        /// <summary> Gets the configuration details for the live model to use </summary>
-        internal Configuration GetConfigurationForLiveModel(string authType, string authValue)
+        /// <summary> Gets the rank process initiated with live model to use </summary>
+        internal RankProcessor GetConfigurationForLiveModel(string authType)
         {
-            _personalizerServiceProperties = ServiceConfigurationRestClient.Get();
-            _personalizerPolicy = PolicyRestClient.Get();
+            liveModelLastRefresh = DateTimeOffset.UtcNow;
```

Comment:
This time should be set after the model has been updated in case an exception is through during update.